.github/workflows/bot.yml

```diff
@@ -74,21 +74,31 @@ jobs:
         env:
           SCALA_PROFILE: ${{ matrix.scalaProfile }}
           SPARK_PROFILE: ${{ matrix.sparkProfile }}
-        run:
-          mvn clean install -D"$SCALA_PROFILE" -D"$SPARK_PROFILE" -pl hudi-examples/hudi-examples-spark,hudi-spark-datasource/hudi-spark -am -DskipTests=true $MVN_ARGS
+        run: >
+          mvnw clean install -pl hudi-examples/hudi-examples-spark,hudi-spark-datasource/hudi-spark -am
+          -D"$SCALA_PROFILE"
```

Comment:
First of all, i think we can't reorder these params as they do have an impact on how goals are executed

But regardless, we'd limit the changes to only swap in mvn for mvnw